R/sce_to_anndata.R

```diff
@@ -0,0 +1,55 @@
+
+#' Convert SingleCellExperiment objects to AnnData file stored as HDF5 file
+#'
+#' @param sce SingleCellExperiment object to be converted to AnnData
+#' @param anndata_file Name of
+#'
+#' @return Converted AnnData object
+#'
+#' @import SingleCellExperiment
+#' @import reticulate
```

Comment:
I'm guessing this is required because we use the `rna_adata$write()` method later? I'd rather avoid it if we can, because importing means it has to be part of the package, I think, and we can't use it as a `Suggests`, which is kind of annoying.